src/main/java/core/basesyntax/service/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.operation.BalanceHandler;
+import core.basesyntax.service.operation.IncreaseHandler;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.PurchaseHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
```

Comment:
```suggestion
public class OperationStrategy {
```
better to use pattern in the class name here